.bingo/Variables.mk

```diff
@@ -23,11 +23,11 @@ $(BINGO): $(BINGO_DIR)/bingo.mod
 	@echo "(re)installing $(GOBIN)/bingo-v0.8.0"
 	@cd $(BINGO_DIR) && GOWORK=off $(GO) build -mod=mod -modfile=bingo.mod -o=$(GOBIN)/bingo-v0.8.0 "github.com/bwplotka/bingo"
 
-CONTROLLER_GEN := $(GOBIN)/controller-gen-v0.14.0
+CONTROLLER_GEN := $(GOBIN)/controller-gen-v0.15.0
```

Comment:
We are using an updated controller gen to generate and test manifests for operator controller, but keeping controller-runtime tied to a previous version. We can either bump controller-gen for o-c in a follow up with others, or do it all here. I would suggest the former, to keep this PR confined to fixing operator sdk with go 1.22 bump.

Reply:
Bumping controller-runtime starts a cascade effect of including k8s v0.30.0, so, yes, I'd prefer to do it later.

Reply:
Sounds good, then we don't need the changes in `.bingo/`. The only change would for controller-gen would be in the set up script that gets used for scaffolding the operator using SDK.

Reply:
The changes in bingo are necessary to install the the correct version of controller-gen, and to install a version of bingo that is compatible with go 1.22.